posthog/helpers/session_recording.py

```diff
@@ -37,6 +37,13 @@ class SnapshotDataTaggedWithWindowId:
     snapshot_data: SnapshotData
 
 
+# NOTE: EventSummary is a minimal version of full events, containing only some of the "data" content - strings and numbers
+class SessionRecordingEventSummary(TypedDict):
+    timestamp: int
+    type: int
+    data: Dict[str, Union[int, str]]
```

Comment:
Could we narrow the str type to a union of some constant strings or otherwise indicate it's related to EVENT_SUMMARY_DATA_INCLUSIONS?

Reply:
I think it would be possible but messy... nested typing in python isn't great. I'll make a comment and see in the bigger PR if it would be possible to type it accurately